docs/1.0. Certificate Provisioning.md

```diff
@@ -179,7 +180,9 @@ It is RECOMMENDED that the EST Server signs the TLS Certificate using an RSA key
 
 ## EST Client
 
-The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that this certificate be valid for a maximum of 10 years. The Root CA used to sign it, is RECOMMENDED to be valid for a maximum of 20 years.
+The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that the client certificate be valid for a maximum of 10 years, however the certificate SHOULD NOT constrain the device's lifetime. The Root CA used to sign the client certificate, is RECOMMENDED to be valid for a maximum of 20 years.
```

Comment:
As discussed, it might be worth using the longer lifetime flagged in the BRSKI draft here, and/or include a suggestion about the EST Server being able to ignore validity dates for manufacturer installed certs (elsewhere in the doc)